src/Microsoft.Security.Utilities.Core/HighPerformanceScanner.Generated.tt

```diff
@@ -0,0 +1,67 @@
+﻿<#@ template debug="true" hostspecific="true" language="C#" #>
+<#@ assembly name="$(TargetPath)" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System" #>
+<#@ import namespace="System.CodeDom.Compiler" #>
+<#@ import namespace="System.Reflection" #>
+<#@ import namespace="Microsoft.Security.Utilities" #>
+<#@ output extension=".cs" #>
+<#@ CleanupBehavior processor="T4VSHost" CleanupAfterProcessingTemplate="true" #>
+// <auto-generated/>
+
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+/*****************************************************************************
+ * WARNING: AUTO-GENERATED. DO NOT MODIFY THE C# SOURCE FILE.
+ *
+ * To regenerate, select the 'Debug' configuration in Visual Studio, open the
+ * .tt file and hit Ctrl+S to save. You can also regenerate by right-clicking on
+ * the .tt file in Solution Explorer and selecting 'Run Custom Tool', or debug
+ * the generation using 'Debug T4 Template'.
+ *
+ * NOTE: This is using T4 instead of a roslyn generator because we must generate
+ * the regular expression constants as input to the the regex source generator.
+ * It does not appear to be possible to chain rolsyn generators in this way.
+ *****************************************************************************/
+
+<#= GenerateCode() #>
+<#+
+private string GenerateCode()
```

Comment:
>GenerateCode [](http://example.com/codeflow?start=15&length=12)

if we're headed this direction we could unify the generated JSON patterns as well. this would eliminate the post-processing step we do today (as a discrete rules export command).

Reply:
#152